vpn/dialer_test.go

```diff
@@ -169,19 +225,10 @@ func makeTestingConnForReadWrite(network, addr string, n int) net.Conn {
 	return mockConn
 }
 
-func makeTestingRawDialer(t *testing.T) RawDialer {
-	raw := RawDialer{
-		Options:       makeTestingOptions(t, "AES-128-GCM", "sha512"),
-		dialer:        &mockedDialerContext{},
-		clientFactory: makeTestingClient,
-	}
-	return raw
-}
-
+// FIXME Dialer needs to be set on the client, it's not on the TunDialer anymore.
```

Comment:
Suggestion: describe this refactoring inside an issue and use `TODO(issue_url)`. You can say this is a bug in the issue tracker. So, we can query easily for TODO and we can easily land to the related issues. Also, we have an uniform syntax to query for what to do and the "color" of the link (chore, bug, enhancement) is in the issue.